protocol/Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG GOLANG_1_22_ALPINE_DIGEST="8e96e6cff6a388c2f70f5f662b64120941fcd7d4b89d62fec
 # Builder
 # --------------------------------------------------------
 
-FROM golang@sha256:${GOLANG_1_22_ALPINE_DIGEST} as builder
```

Comment:
- GOLANG_1_22_ALPINE_DIGEST is not used anywhere else
- Should still use a var instead of hard coding this value

Reply:
Good point.